book/_toc.yml

```diff
@@ -78,15 +69,6 @@ parts:
           title: Accessing SnowEx Data at the NSIDC DAAC
           sections:
             - file: tutorials/nsidc-data-access/nsidc-data-access
-        - file: tutorials/microwave/index
-          title: Microwave Tutorials
-          sections:
-            - file: tutorials/microwave/swesarr/swesarr_tut
-              title: SWESARR
-        - file: tutorials/aviris-ng/index
-          title: AVIRIS-NG
-          sections:
-            - file: tutorials/aviris-ng/AVIRIS-NG_Tutorial
```

Comment:
see above, also need to add these back in

Reply:
got it, doing that now. also running these out of the hackweek .yml now so that shouldn't be an issue